src/Symfony/Component/HttpClient/HttplugClient.php

```diff
@@ -51,7 +51,7 @@
 }
 
 /**
- * An adapter to turn a Symfony HttpClientInterface into an Httplug client.
+ * An adapter to turn a Symfony HttpClientInterface into a Httplug client.
```

Comment:
This is actually ok because it begins with a vowel sound.

Reply:
Im a native English speaker since birth... Im not going to argue but I think this is borderline and :) reverted.

Comparing google search result numbers for "a http client" and "an http client" you win by 100.000+ :)

Reply:
Yeah, there doesn't seem to be a consensus when googling, I was actually using the docs as a guide line. There it's "an HTML page" or "an HTTP request", so at least it's consistent this way.

Reply:
its all cool :) no argument here.